templates/desktop/developers.html

```diff
@@ -26,228 +26,562 @@ <h1>Ubuntu Desktop for&nbsp;Linux developers</h1>
+                ) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/cd765ccc-nvidia-logo.png",
+                alt="Nvidia",
+                width="288",
+                height="288",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-logo-section__logo"}
+                ) | safe
+              }}
+            </div>
+          </div>
+        </div>
       </div>
-      <div class="col-5 u-vertically-center">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/8c02c97a-Laptop-X1-Carbon.png",
-          alt="",
-          width="622",
-          height="489",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
+      <div class="col-6">
+        <h3>Certified hardware</h3>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Streamline deployment of Ubuntu Desktop with <a href="/certified">certified hardware</a> from OEMs including Dell, Lenovo, and HP &mdash; no additional configuration required.</li>
+          <li class="p-list__item is-ticked">Ubuntu performs out of the box across a wide range of laptops, desktops, and workstations, including the HP Z series and Dell  XPS and Precision lines.</li>
+          <li class="p-list__item is-ticked"><a href="/desktop/partners">Strategic partnerships</a> extend to component manufacturers and Ubuntu is certified to work with the full range of NVIDIA GPUs.</li>
+       </ul>
+       <p>
+         <a href="/certified">Ubuntu certified hardware&nbsp;&rsaquo;</a>
+       </p>
       </div>
     </div>
   </section>
 
-  <section class="p-strip--suru-accent is-dark">
+  <section class="p-strip is-bordered">
     <div class="u-fixed-width">
-      <blockquote class="p-pull-quote">
-        <p class="p-pull-quote__quote">Ubuntu has been the perfect OS given its popularity with developers and its cloud capabilities. That's why it's preloaded on the 4th generation of our XPS 13 laptop and our new Precision M3800 Mobile Workstation.</p>
-        <cite class="p-pull-quote__citation">Barton George, Director of developer programs at Dell services</cite>
-      </blockquote>
-      <p><a href="/blog/designed-for-developers-dell-launches-two-new-ubuntu-based-systems" class="p-link--inverted">Learn more about Dell's developer edition line</a></p>
+      <h2 class="u-sv3">The preferred OS for developers and organisations</h2>
     </div>
-  </section>
-
-  <section class="p-strip">
-    <div class="row u-equal-height u-vertically-center u-hide--small u-hide--medium">
-      <div class="col-5">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/bda34a6e-wordpress-apache-mysql-bundle-trans.svg",
+    <div class="row">
+      <div class="col-6">
+        <h3 class="u-sv2 p-heading--4">Operating systems developers prefer</h3>
+        <p>Multiple choice poll<p>
```

Comment:
I think this can be hidden on small and medium screen as it doesn't make sense without the chart.

Reply:
Could I hide all in this section?